src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxdatainterface.py

```diff
@@ -35,11 +35,12 @@ def add_recording_extractor_properties(recording_extractor) -> None:
 
 
 class SpikeGLXRecordingInterface(BaseRecordingExtractorInterface):
-    help = "Interface for SpikeGLX recording data."
     display_name = "SpikeGLX Recording"
     keywords = BaseRecordingExtractorInterface.keywords + [
         "Neuropixels",
     ]
+    common_suffixes = ["^.imec\d+$", ".ap", ".lf", ".bin"]
```

Comment:
These could either be regex patterns in the case of suffix patterns like `.imec0`, `.imec1`, etc. here - or just static strings